lib/sensors/i2c_mux.hpp

```diff
@@ -0,0 +1,133 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <array>
+#include <memory>
+
```

Comment:
Check your includes here. Missing at least cstdint, optional and core/types.hpp.

Reply:
coming from i2c_sensors - but yeah will add jic